lib/config/definitions.ts

```diff
@@ -1903,6 +1903,19 @@ const options: RenovateOptions[] = [
     mergeable: true,
     cli: false,
   },
+  {
+    name: 'cocoapods',
+    releaseStatus: 'beta',
+    description: 'Configuration object for Podfiles',
+    stage: 'repository',
+    type: 'object',
+    default: {
+      enabled: false,
+      fileMatch: ['(^|/)Podfile$'],
+      versionScheme: 'ruby',
```

Comment:
Does it support the full syntax of Ruby, particularly ranges?

Reply:
I think yes, as far as `ruby` versioning does: [example PR](https://github.com/zharinov/FireSnapshot-3/pull/1).